resources/packs/ms365/ms365.lr

```diff
@@ -146,46 +144,41 @@ msgraph.beta.rolemanagement.roledefinition {
   rolePermissions []dict
   templateId string
   version string
-  assignments() []msgraph.beta.rolemanagement.roleassignment
+  assignments() []msgraph.rolemanagement.roleassignment
 }
 
-// Microsoft Graph Beta Role Assignment
-msgraph.beta.rolemanagement.roleassignment {
+// Microsoft Graph Role Assignment
+msgraph.rolemanagement.roleassignment {
   id string
   roleDefinitionId string
   principalId string
   principal dict
 }
 
-// Microsoft Graph Beta Device Management
-msgraph.beta.devicemanagement {
-  deviceConfigurations() []msgraph.beta.devicemanagement.deviceconfiguration
-  deviceCompliancePolicies() []msgraph.beta.devicemanagement.devicecompliancepolicy
+// Microsoft Graph Device Management
+msgraph.devicemanagement {
+  deviceConfigurations() []msgraph.devicemanagement.deviceconfiguration
+  deviceCompliancePolicies() []msgraph.devicemanagement.devicecompliancepolicy
 }
 
-// Microsoft Graph Beta Device Configuration
-msgraph.beta.devicemanagement.deviceconfiguration {
+// Microsoft Graph Device Configuration
+msgraph.devicemanagement.deviceconfiguration {
   id string
   lastModifiedDateTime time
-  roleScopeTagIds []string
-  supportsScopeTags bool
   createdDateTime time
   description string
   displayName string
   version int
-  properties dict
 }
 
-// Microsoft Graph Beta Device Compliance Policy
-msgraph.beta.devicemanagement.devicecompliancepolicy {
+// Microsoft Graph Device Compliance Policy
+msgraph.devicemanagement.devicecompliancepolicy {
   id string
   createdDateTime time
   description string
   displayName string
   lastModifiedDateTime time
-  roleScopeTagIds []string
   version int
-  properties dict
```

Comment:
I have removed the `properties` from most of the objects. The MSGraph API does not really have properties-like field as everything is exposed as a separete field there (which makes sense since they use odata).

I think this also is a step in the right direction in general - makes writing policies/queries easier as we have a better idea of what the resource contains just by looking at its definition. Else, one would have to see what the `properties` dict contains and then assert against it.

@chris-rock would love your thoughts on this in general

Reply:
Throwing an error is the right thing to do. We do not want to pretend the test worked. For old policies, this gives the users a proper error.